ratpack-core/src/main/java/ratpack/server/internal/ServerRegistry.java

```diff
@@ -58,6 +62,16 @@
 import static ratpack.util.internal.ProtocolUtil.HTTP_SCHEME;
 
 public abstract class ServerRegistry {
+
+  public static AddressResolverGroup<?> defaultAddressResolver(ExecController controller) {
+    DnsNameResolverBuilder resolverBuilder = new DnsNameResolverBuilder()
+      .eventLoop(controller.getEventLoopGroup().next())
```

Comment:
This does mean that all DNS resolves are bound to 1 event loop...which, since it's NIO _shouldn't_ be a problem (and it is how `reactor-netty` does it too).